docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -89,11 +151,13 @@ spec:
         imagePullPolicy: Always
         env:
         - name: CONTAINER_TOKEN
-          value: <YOUR_DOCKER_APP_TOKEN_HERE>
+          value: <YOUR_CONTAINER_TOKEN>
         - name: INFRA_TOKEN
-          value: <YOUR_INFRA_APP_TOKEN_HERE>
-        - name: REGION:
-          value: <US or EU>
+          value: <YOUR_INFRA_TOKEN>
+        - name: LOGS_TOKEN
+          value: <YOUR_LOGS_TOKEN>
+        - name: REGION
+          value: "US"
```

Comment:
Can we use <"US" or "EU"> so this is self-documenting?  People will need to add their tokens anyway, even if they copy-paste this.